pwndbg/gdblib/disasm/instruction.py

```diff
@@ -248,6 +247,28 @@ def __init__(self, cs_insn: CsInsn | None) -> None:
         None if we don't have a determination (most cases)
         """
 
+        self.declare_is_unconditional_jump: bool = False
+        """
+        This field is used to declare that this instruction is an unconditional jump.
+        Most of the type, we depend on Capstone groups to check for jump instructions,
```

Comment:
"Most of the type"? Shall this be one of:
- "Most of the time"
- "For most of instruction types"
- "For most of instructions"
?